third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -5111,7 +5111,20 @@ def replace(
 
     @property
     def iloc(self):
-        """Purely integer-location based indexing for selection by position."""
+        """Purely integer-location based indexing for selection by position.
+
+        Returns:
+            ILocDataFrameIndexer: Purely integer-location Indexers.
```

Comment:
I think this will need to be the fully qualified name so that it renders as a link. Same for the others. See internal issue 296390934.

```suggestion
            bigframes.core.indexers.ILocDataFrameIndexer:
                Purely integer-location Indexers.
```